src/current/v24.1/cutover-replication.md

```diff
@@ -73,24 +74,23 @@ To select a [specific time]({% link {{ page.version.version }}/as-of-system-time
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-SHOW VIRTUAL CLUSTER application WITH REPLICATION STATUS;
+SHOW VIRTUAL CLUSTER main WITH REPLICATION STATUS;
 ~~~
 
 The `retained_time` response provides the earliest time to which you can cut over.
 
-{% include_cached copy-clipboard.html %}
 ~~~
-id |        name        |     data_state     | service_mode | source_tenant_name |                                                     source_cluster_uri                                               | replication_job_id |        replicated_time        |         retained_time         | cutover_time
----+--------------------+--------------------+--------------+--------------------+----------------------------------------------------------------------------------------------------------------------+--------------------+-------------------------------+-------------------------------+---------------
-3  | application        | replicating        | none         | application        | postgresql://{user}:redacted@{hostname}:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted   | 899090689449132033 | 2023-09-11 22:29:35.085548+00 | 2023-09-11 16:51:43.612846+00 |     NULL
+  id | name | source_tenant_name |              source_cluster_uri               |         retained_time         |    replicated_time     | replication_lag | cutover_time |   status
```

Comment:
The column headers do not match with the data, so it is hard to know what is the value for `retained_time`. Please align the headers with the data.

Reply:
fixed